webview/src/util/wdyr.ts

```diff
@@ -5,6 +5,6 @@ import React from 'react'
 if (process.env.NODE_ENV === 'development') {
   const whyDidYouRender = require('@welldone-software/why-did-you-render')
   whyDidYouRender(React, {
-    trackAllPureComponents: true
+    trackAllPureComponents: false
```

Comment:
WDYR is still a useful tool, but having this set to true was getting quite annoying.